src/pages/Login/container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+
+import Login from './component';
+import { sessionSetId } from '../../../state/session/actions';
+
+const mapDispatchToProps = (dispatch) => ({
```

Comment:
You can simplify this method like:
```
const mapDispatchToProps = {
  sessionSetId
}
```

which automatically wrap this function with `dispatch`

Reply:
Done.